docs/manual_doc/Language_Structure/.User-Defined Variables.md

```diff
@@ -0,0 +1,16 @@
+>User-Defined Variable can only be used in one excution ( coupound statements separated by semicolon (;) or pipe (|) and be submitted to server to excute together)
+
+<!-
```

Comment:
It seems to be a problem with the comment's format.

Please see [Here](https://github.com/vesoft-inc/nebula/blob/14ac4625b96e638c198d54c01b9317cc2b9c6d9d/docs/manual_doc/Language_Structure/.User-Defined%20Variables.md)